File: LISTAS-aeds1/AreaQuadrado/AreaQuadrado.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
    float lado;
    printf("Digite o lado do quadrado:   ");
    
    scanf("%f", &lado); 
    
    float resultado = lado * lado;
    
    printf("A area do quadrado e: %f", resultado); 
    
    return 0; 
}

File: LISTAS-aeds1/AreaQuadrado/AreaQuadrado.java
import java.util.*;

// FUNÇÃO JAVA PARA CALCULAR A AREA DE UM QUADRADO QUALQUER//
// FORMULA PARA O CALCULO: Area = lado x lado; //
class AreaQuadrado
{
    //Leitura e formula da função//
    public static void main(String[] args)
    {
        Scanner sc = new Scanner(System.in);
        // Tratamento de exeção para lidar com numeros negativos//
        try 
        {
            System.out.println("Digite o lado do quadrado: ");
            double lado = sc.nextDouble();
            if(lado < 0)
            {
                throw new IllegalArgumentException("Voce deve digitar um numero valido");
            }
            double resultado;
            resultado = lado * lado;
            System.out.println("A area do quadrado e: " + resultado);
        }
        catch (IllegalArgumentException e) {
            // e.getMessage() recupera a mensagem que definimos no "throw".
            System.out.println("ERRO: " + e.getMessage());
        } 
    
    }
}